src/migrations/migration.js

```diff
@@ -0,0 +1,40 @@
+const createUser = `
```

Comment:
Use more descriptive variable names. This one should be `createUserTableQuery`, and so on.